src/test/java/uk/gov/justice/probation/courtcaseservice/pact/CourtCaseMatcherVerificationPactTest.java

```diff
@@ -207,6 +210,6 @@ void getCaseByHearingId() {
     @State({"a case will be PUT by id"})
     void mockPutCourtCaseExtended() {
         final Mono<HearingEntity> caseMono = Mono.just(EntityHelper.aHearingEntity("X340741", "1600028914"));
-        when(courtCaseService.createHearing(eq("D517D32D-3C80-41E8-846E-D274DC2B94A5"), any(HearingEntity.class))).thenReturn(caseMono);
+        when(courtCaseService.createHearing(eq("ABCDD32D-3C80-41E8-846E-D274DC2B94A5"), any(HearingEntity.class))).thenReturn(caseMono);
```

Comment:
I think this should now be mocking the `createHearingByHearingId` method, think that might be why the tests are still failing